docs/architecture/sdk/password-manager/web/interoperability.md

```diff
@@ -0,0 +1,270 @@
+# Interoperability
+
+When working with the SDK, one of the key challenges is managing Rust/JavaScript bindings to enable
+smooth interactions. In our project, we use both `wasm-bindgen` and `tsify` to bridge this gap and
+make it easier for JavaScript to work with Rust code. While both tools facilitate communication
+between Rust and JavaScript, they serve distinct purposes, and it's important to understand when to
+use each.
+
+## `wasm-bindgen` vs. `tsify`
+
+:::tip
+
+**In short:** Use `tsify` unless the web-side needs to call functions or interact with Rust objects
```

Comment:
For the future, you can assign a custom title to admonitions, https://docusaurus.io/docs/markdown-features/admonitions#specifying-title